tests/benchmarks/power_consumption/temperature_sensor/testcase.yaml

```diff
@@ -0,0 +1,22 @@
+common:
+  sysbuild: true
+  tags:
+    - drivers
+    - sensors
+    - ci_tests_drivers_sensor
```

Comment:
Please use tag `ci_tests_benchmarks_current_consumption`
instead of `ci_tests_drivers_sensor` - this is assigned to completely different tests and assignment is base on folder/path, not the features.